Testscripts/Linux/validate-da.sh

```diff
@@ -0,0 +1,163 @@
+#!/usr/bin/env bash
+#######################################################################
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License.
+#
+# validate-da.sh
+# Description:
+#    Validate the installation and uninstallation of the Dependency Agent.
+#    Validate the enable and disable of the Dependency Agent.
+#######################################################################
+
+set -e
```

Comment:
Based upon @LiliDeng's comments below about not exiting with a non-zero exit status, is setting the `-e` option appropriate since it will cause a non-zero exit from this shell for any unexpected command failure?

Reply:
We experiment the line number 12 and 13 do not work in LISAv2 run-time. @rakshith210 Do you find any reason or are you going to take out?

Reply:
I have take both out as part of my latest iteration